imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -205,14 +200,38 @@ class OrderTable extends Component {
     if (this.props.isOpen) {
       // Render order list column/row data
       const filteredFields = {
-        "Name": "shipping[0].address.fullName",
-        "Email": "email",
-        "Date": "createdAt",
-        "ID": "_id",
-        "Total": "billing[0].invoice.total",
-        "Shipping": "shipping[0].workflow.status",
-        "Status": "workflow.status",
-        "": ""
+        "Name": {
+          accessor: row => getShippingInfo(row).address.fullName,
+          id: "shippingfullName"
+        },
+        "Email": {
+          accessor: "email",
+          id: "email"
+        },
+        "Date": {
+          accessor: "createdAt",
+          id: "createdAt"
+        },
+        "ID": {
+          accessor: "_id",
+          id: "_id"
+        },
+        "Total": {
+          accessor: row => getBillingInfo(row).invoice.total,
+          id: "billingTotal"
+        },
+        "Shipping": {
+          accessor: row => getShippingInfo(row).workflow.status,
+          id: "shippingStatus"
+        },
+        "Status": {
+          accessor: "workflow.status",
+          id: "workflow.status"
+        },
+        "": {
```

Comment:
Can this empty string object key actually work?  I know it was already there, but I'm unsure how that didn't cause a problem before.

Reply:
@jshimko yeah, it works. It creates a column with no header/label. That is the last column on the order table. But I can remove the empty string key/replace it with an actual string and find a workaround for creating the headerless column.